style.css

```diff
@@ -0,0 +1,187 @@
+/* Media Queries*/
+@media only screen(max-width:500px) {
+    #sidebar{
```

Comment:
# css
- Indentation is needed after the #sidebar before the curly brace in line 3